CS483/CS483/Kartaclysm/Data/UserConfig/ControlBindings.xml

```diff
@@ -1,54 +1,54 @@
 <ControlBindings>
   <Keyboard1>
-      <accelerate value="265" />
-      <brake value="264" />
-      <left value="263" />
-      <right value="262" />
-      <slide value="32" />
-      <driverAbility1 value="65" />
-      <driverAbility2 value="83" />
-      <kartAbility1 value="68" />
+      <accelerate value="87" />
+      <brake value="83" />
+      <left value="65" />
+      <right value="68" />
+      <slide value="90" />
+      <driverAbility1 value="81" />
+      <driverAbility2 value="69" />
+      <kartAbility1 value="82" />
       <kartAbility2 value="70" />
-      <pause value="257" />
+      <pause value="88" />
```

Comment:
What I mean is that Enter used to be the pause button for player 1, but now it's X. InputActionMapping (line 331,342, etc.) still reads the default as:
(*m_pInputMap)[Input::eKeyboard1][Racer::ePause] = GLFW_KEY_ENTER;
(*m_pInputMap)[Input::eKeyboard2][Racer::ePause] = GLFW_KEY_ENTER;